lib/chef/provisioning/machine/unix_machine.rb

```diff
@@ -96,6 +99,9 @@ def set_attributes(action_handler, path, attributes)
       # Get file attributes { :mode, :owner, :group }
       def get_attributes(path)
         result = transport.execute("stat -c '%a %U %G %n' #{path}", :read_only => true)
+        unless result.exitstatus == 0
+          result = transport.execute("stat  -f '%OLp %Sg %Su %N' #{path}", :read_only => true)
```

Comment:
Double space `stat  -f` and switch positions of `%Sg %Su` => `%Su %Sg` to be consistent with `%U %G`